DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -212,24 +221,98 @@ public void PromoteDelegateToAdmin(AdminRoles adminRoles, int categoryId, int de
                     delegateUser.ProfessionalRegistrationNumber,
                     categoryId,
                     adminRoles.IsCentreAdmin,
-                    false,
+                    adminRoles.IsCentreManager,
                     adminRoles.IsSupervisor,
                     adminRoles.IsNominatedSupervisor,
                     adminRoles.IsTrainer,
                     adminRoles.IsContentCreator,
                     adminRoles.IsCmsAdministrator,
                     adminRoles.IsCmsManager,
+                    supervisorDelegateUser.Id,
+                    supervisorDelegateUser.EmailAddress ?? string.Empty,
+                    supervisorDelegateUser.FirstName ?? string.Empty,
+                    supervisorDelegateUser.LastName,
                     delegateUser.ProfileImage
                 );
 
                 registrationDataService.RegisterAdmin(adminRegistrationModel, false);
+                SendEmailNotification(adminRegistrationModel);
             }
             else
             {
                 throw new AdminCreationFailedException(AdminCreationError.EmailAlreadyInUse);
             }
         }
 
+        private void SendEmailNotification(AdminRegistrationModel adminRegistrationModel)
+        {
+            var emailSubjectLine = "New Digital Learning Solutions permissions granted";
+
+            var builder = new BodyBuilder();
+
+            builder.TextBody = $@"Dear {adminRegistrationModel.FirstName},
+                                The user {adminRegistrationModel.SupervisorFirstName} {adminRegistrationModel.SupervisorLastName} has granted you new access permissions in the Digital Learning Solutions system.
+                                You have been granted the following permissions:";

```

Comment:
Tweaked this line to make read better when only one permission has been granted.